app/managers.py

```diff
@@ -0,0 +1,46 @@
+from sqlite3 import Connection, connect
+from entity_helper import EntityHelper
+from models import Actor
+from os import path
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        db_path = path.join(path.dirname(path.abspath(__file__)), "cinema.db3")
+        self.__connector: Connection = connect(db_path)
+        self.__actor_helper: EntityHelper = EntityHelper(Actor, "actors")
+
+    def __del__(self) -> None:
+        if self.__connector:
+            self.__connector.close()
+
+    def create(self, actor: Actor) -> None:
+        cursor = self.__connector.execute(
+            self.__actor_helper.get_insert_statement_param(),
```

Comment:
Why don't you want to put f-string here directly instead of calling a function, which will just give you an attribute of __actor_helper?

Reply:
@LiudmylaKulinchenko  Because I want to stay consistent in my mapper implementation. I could create properties for almost all of those statements. All, but the update statement, since it can not simply be parametrized with a template just like other sql commands. I don't like an idea of making some of object's attributes accessible by properties and others - by methods. The only reasonable solution of your request here is to implement another layer of abstraction so I could hide those implementation details withing some sort of a `data set` class as a Repository/Unit of work subsystem, and that is something that is far outside of a scope of responsibilities for that EntityHelper class, which is, essentially, a low level mapper. 

All in all, I want to keep this design as it is